testing/integration_testing/requirements.txt

```diff
@@ -0,0 +1,2 @@
+absl-py
```

Comment:
Should we name this file `pip_requirements.txt` to be clearer on what it is?

Reply:
From what I can tell, it's a soft convention in the Python community to call it requirements.txt.

Reply:
Sure, we can leave it as it is. In my case, there are a bunch of these files for vcpkg too and am using this naming scheme it to distinguish between them. (vcpkg_* vs pip_*).